src/composables/protocol.ts

```diff
@@ -0,0 +1,56 @@
+import type {
+  Class,
+  IDefaultComposableOptions,
+  Protocol,
+} from '@/types';
+import { BaseProtocol } from '@/protocols/BaseProtocol';
+import { AeternityProtocol } from '@/protocols/aeternity/libs/AeternityProtocol';
+import { BitcoinProtocol } from '@/protocols/bitcoin/libs/BitcoinProtocol';
+import {
+  PROTOCOL_AETERNITY,
+  PROTOCOL_BITCOIN,
+} from '@/constants';
+import { watchUntilTruthy } from '@/utils';
+import { useAccounts } from './accounts';
+
+let isProtocolRegistryReady = true;
```

Comment:
Why we need this? The protocol classes are synchronous so no waiting is required.
Also what actually means "protocol registry ready"? When it's full, or at least one protocol is instantiated?

Reply:
No longer needed